packages/adapter-netlify/index.js

```diff
@@ -47,11 +54,20 @@ export default function ({ split = false, edge = edge_set_in_env_var } = {}) {
 			// empty out existing build directories
 			builder.rimraf(publish);
 			builder.rimraf('.netlify/edge-functions');
-			builder.rimraf('.netlify/functions-internal');
 			builder.rimraf('.netlify/server');
 			builder.rimraf('.netlify/package.json');
 			builder.rimraf('.netlify/serverless.js');
 
+			const functions_internal = join('.netlify', 'functions-internal');
```

Comment:
this could probably use a comment for the future
```suggestion
			// don't delete the entire directory because `netlify build` may put stuff there
			// just clear the files we've generated instead
			const functions_internal = join('.netlify', 'functions-internal');
```

Reply:
That makes sense. Let me rephrase that suggestion slightly